keyserver/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=1.2.8
```

Comment:
Stupid question: why is the SBT version defined in each sub-project? Can we just leave that out?

Reply:
Good point. I tried removing the ones in the subprojects and it doesn't seem to make a difference. So I can amend the commit and remove them, unless @simao you think there's a reason to keep them?

Reply:
Lets use just one if that works.